Update README.md

ArthurP2001 opened the pull request:
remise

ArthurP2001 committed:
Update README.md

remise

bbaudry commented:
@ArthurP2001 : ta pull request est en conflit et ne peut pas être 'merge'. Tu dois soumettre une PR mise à jour, sans conflite, dans les 24 heures: avant mercredi 15 octobre 15.30
Cordialement,
Benoit Baudry

ArthurP2001 commented:
Bonjour @bbaudry ,
Ma pull request est en conflit  avec quoi exactement?

Est ce qu elle est en conflit avec celle de ma coéquipière?

Est ce que pour resoudre le problème je dois juste refaire une pull request
et ce sera correct? Oubien je dois faire un truc en plus

On Tue, Oct 14, 2025 at 3:19 PM Benoit Baudry ***@***.***>
wrote:

> *bbaudry* left a comment (umontreal-diro/IFT3913#711)
> <https://github.com/umontreal-diro/IFT3913/pull/711#issuecomment-3403255968>
>
> @ArthurP2001 <https://github.com/ArthurP2001> : ta pull request est en
> conflit et ne peut pas être 'merge'. Tu dois soumettre une PR mise à jour,
> sans conflite, dans les 24 heures: avant mercredi 15 octobre 15.30
> Cordialement,
> Benoit Baudry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/umontreal-diro/IFT3913/pull/711#issuecomment-3403255968>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDYGSOD6WZJFVCRGN66S75L3XVEDXAVCNFSM6AAAAACIZGSHNKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTIMBTGI2TKOJWHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ArthurP2001 committed:
Remise TP1 - Conflit PR

just received a message from the teacher telling me to resend a PR because me previous one 5 days ago was In conflict.

bbaudry commented:
@ArthurP2001 : tu peux voir la source du conflit en analysant le contenu de "This branch has conflicts that must be resolved"

ArthurP2001 committed:
Merge branch '2025' into patch-5

ArthurP2001 commented:
@bbaudry 
Bonjour,
J’ai résolu le conflit and marked it as solved. I recommitted the file directly inside the branch.
